### PR TITLE
enh(mc): automatic commit for network-cisco-wap-snmp

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md [hunk 1]
```diff
@@ -5,82 +5,118 @@ title: Cisco Wap SNMP
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Contenu du pack
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md:8]:
```suggestion
Cisco est un fournisseur de solutions réseau.

Le connecteur de supervision *Cisco Wap SNMP* utilise le protocole SNMP pour se connecter, récupérer des informations et les métriques relatives aux point d'accès de marque Cisco.

## Contenu du pack
```

File: pp/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md [hunk 2]
```diff
@@ -5,82 +5,117 @@ title: Cisco Wap SNMP
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Pack assets
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md:8]:
```suggestion
Cisco  is a provider of network solutions.

The Centreon Monitoring Connector *Cisco Wap SNMP* relies on the SNMP protocol to query and collect the status and metrics of the Cisco Wireless Access Point equipment.

## Pack assets
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md [hunk 3]
```diff
@@ -5,82 +5,122 @@ title: Cisco Wap SNMP
...
-<TabItem value="Clients-1" label="Clients">
+<TabItem value="Net-Cisco-Wap-SNMP-custom" label="Net-Cisco-Wap-SNMP-custom">
+
+| Alias                 | Modèle de service                              | Description                                                 |
+|:----------------------|:-----------------------------------------------|:------------------------------------------------------------|
+| Clients               | Net-Cisco-Wap-Clients-SNMP-custom              | Contrôle le nombre de clients connectés                     |
+| Cpu                   | Net-Cisco-Wap-Cpu-SNMP-custom                  | Contrôle le taux d'utilisation du CPU                       |
+| Memory                | Net-Cisco-Wap-Memory-SNMP-custom               | Contrôle du taux d'utilisation de la mémoire vive           |
+| Uptime                | Net-Cisco-Wap-Uptime-SNMP-custom               | Durée depuis laquelle l'équipement tourne sans interruption |
+| Virtual Access Points | Net-Cisco-Wap-Virtual-Access-Point-SNMP-custom | Contrôle les points d'accès virtuel                         |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md:32]:
```suggestion
| Virtual Access Points | Net-Cisco-Wap-Virtual-Access-Point-SNMP-custom | Contrôle les points d'accès virtuels                         |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md [hunk 4]
```diff
@@ -5,82 +5,122 @@ title: Cisco Wap SNMP
...
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de services
 
-| Metric name                         | Description                    | Unit   |
-| :---------------------------------- | :----------------------------- | :----- |
-| cpu.utilization.percentage          | Percentage of CPU utilization  | %      |
+| Nom de la règle                   | Description                                                             |
+|:----------------------------------|:------------------------------------------------------------------------|
+| Net-Cisco-Wap-SNMP-Interface-Name | Découvre les interfaces réseaux et supervise le statut et l'utilisation |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md:64]:
```suggestion
| Net-Cisco-Wap-SNMP-Interface-Name | Découvre les interfaces réseau et supervise le statut et l'utilisation |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md [hunk 5]
```diff
@@ -5,82 +5,121 @@ title: Cisco Wap SNMP
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-
-## Overview
-
 Cisco  is a provider of network solutions.
 
 The Centreon Monitoring Connector *Cisco Wap SNMP* relies on the SNMP protocol to query and collect the status and metrics of the Cisco Wireless Access Point equipment.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md:10]:
```suggestion
The Centreon Monitoring Connector *Cisco Wap SNMP* relies on the SNMP protocol to query and collect the status and metrics of Cisco Wireless Access Point devices.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md [hunk 6]
```diff
@@ -5,82 +5,122 @@ title: Cisco Wap SNMP
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-
-## Vue d'ensemble
-
 Cisco est un fournisseur de solutions réseau.
 
 Le connecteur de supervision *Cisco Wap SNMP* utilise le protocole SNMP pour se connecter, récupérer des informations et les métriques relatives aux point d'accès de marque Cisco.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md:10]: c'est des appareils qui s'appellent Wireless Access Point? (cf version anglaise)

File: pp/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md [hunk 7]
```diff
@@ -90,100 +129,437 @@ The Centreon Monitoring Connector *Cisco Wap SNMP* relies on the SNMP protocol t
 The SNMP agent must be configured and running. Please refer to the manufacturer documentation to achieve this.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md:129]: 1. I think this is not the standard text, is it?
2. FR and EN are different

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md (hunk 7 above)
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md:129]:
```suggestion
The SNMP service must be configured and activated on the host. Please refer to the official documentation from the manufacturer/publisher.

```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md (hunk 6 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md:10]:
```suggestion
Le connecteur de supervision *Cisco WAP SNMP* utilise le protocole SNMP pour se connecter, récupérer des informations et les métriques relatives aux point d'accès de marque Cisco.
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**